From 205aab4465e7c68c829699fdc15db1f225917a66 Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Thu, 26 Jun 2014 13:58:58 +0800
Subject: [PATCH 0431/1074] ENGR00320056-4 ARM: dts: imx6sx-19x19-arm2: Add
 CSI2 support

Add CSI2 support.
use CSI2 for VADC instead of CSI1.

Signed-off-by: Robby Cai <r63905@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6sx-19x19-arm2.dts |   13 ++++++++++++-
 1 files changed, 12 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-19x19-arm2.dts b/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
index 3287d71..9eb6b95 100644
--- a/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
@@ -23,6 +23,13 @@
 
 	csi1_v4l2_cap {
 		compatible = "fsl,imx6sx-csi-v4l2", "fsl,imx6sl-csi-v4l2";
+		csi_id = <0>;
+		status = "okay";
+	};
+
+	csi2_v4l2_cap {
+		compatible = "fsl,imx6sx-csi-v4l2", "fsl,imx6sl-csi-v4l2";
+		csi_id = <1>;
 		status = "okay";
 	};
 
@@ -99,6 +106,10 @@
 	status = "okay";
 };
 
+&csi2 {
+	status = "okay";
+};
+
 &fec1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_enet1_1>;
@@ -492,6 +503,6 @@
 
 &vadc {
 	vadc_in = <0>;
-	csi_id = <0>;
+	csi_id = <1>;
 	status = "okay";
 };
-- 
1.7.5.4

